docs/source/conf.py

```diff
@@ -19,8 +19,9 @@
 import os
 import subprocess
 from recommonmark.parser import CommonMarkParser
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import sys
+python_libs = os.path.abspath('../../lib/python')
```

Comment:
this is not part of this PR but in #2726 and will be rebased out.
Our docs are build in `<repo>/docs` always (and that workflow is documented), so that is ok for now.